debian/ubuntu-advantage-tools.maintscript

```diff
@@ -2,3 +2,4 @@ rm_conffile /etc/update-motd.d/99-esm 19.1~ ubuntu-advantage-tools
 rm_conffile /etc/update-motd.d/80-esm 19.1~ ubuntu-advantage-tools
 rm_conffile /etc/update-motd.d/80-livepatch 19.1~ ubuntu-advantage-tools
 rm_conffile /etc/cron.daily/ubuntu-advantage-tools 19.1~ ubuntu-advantage-tools
+rm_conffile /etc/init/ua-auto-attach.conf 20.2~ ubuntu-advantage-tools
```

Comment:
This is correct for focal, but what about for trusty?  what is the version number that will be uploaded to trusty that doesn't also match this version?  (I ask, because debian/rules includes per-series handling, but there is no per-series handling in the .maintscript file.)

Reply:
@vorlonofportland, thanks for the review here. For the only public Pro images on trusty, the package version on trusty is 19.7-0~201911281740~ubuntu14.04.1.

There will be a second release of pro images that will contain 20.1* for Trusty Xenial and Bionic. We need to support migration from either 19.7 or 20.1 major.minor versions as viable upgrade paths and drop the init script or systemd units appropriately across that upgrade